src/main/java/randoop/generation/AbstractGenerator.java

```diff
@@ -331,6 +331,10 @@ public void createAndClassifySequences() {
           outErrorSeqs.add(eSeq);
         } else {
           outRegressionSeqs.add(eSeq);
+
+          // Bloodhound uses this hook to increment the number of times the last operation
```

Comment:
I don't think this comment belongs in AbstractGenerator.  Maybe in Bloodhound's implementation istead?